flink-runtime/src/main/java/org/apache/flink/runtime/taskexecutor/TaskExecutor.java

```diff
@@ -292,6 +292,13 @@ public void start() throws Exception {
 			throwable = ExceptionUtils.firstOrSuppressed(t, throwable);
 		}
 
+		try {
+			// it will call close() recursively from the parent to children
+			taskManagerMetricGroup.close();
```

Comment:
this method never throws exceptions

Reply:
I was intended to catch maybe some `RuntimeException`... I will just remove the `try catch`.